plugins/lxd.plugin

```diff
@@ -48,7 +48,18 @@ EOF
 
   mv $WORK_DIR/root $WORK_DIR/rootfs
 
-  tar $TAR_PXZ -cf $WORK_DIR/${deployment} -C $WORK_DIR metadata.yaml templates rootfs
+  case $DEPLOYMENT_COMPRESSION in
+      none)
+         tar -cf $WORK_DIR/$deployment -C $WORK_DIR metadata.yaml templates rootfs
+      ;;
+      xz)
+         tar $TAR_PXZ -cJf $WORK_DIR/$deployment -C $WORK_DIR metadata.yaml templates rootfs
```

Comment:
This doesn't work on stretch as TAR_PXZ is set to `pxz` but `J` option is used for xz.